keyboards/ymd75/rev2/rules.mk

```diff
@@ -0,0 +1,28 @@
+# MCU name
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = bootloadHID
```

Comment:
This stuff can stay in the top-level rules.mk (at least until YMDK make a rev3 or something...)